src/sentry/tasks/backfill_seer_grouping_records.py

```diff
@@ -108,13 +114,19 @@ def backfill_seer_grouping_records(
         )
         return
 
-    group_id_message_batch = {
+    group_id_message_batch_filtered = {
         group_id: message
         for (group_id, message, data) in group_id_message_data_batch
-        if not get_path(data, "metadata", "embeddings_info", "nn_model_version")
+        if get_path(data, "metadata", "embeddings_info", "nn_model_version") is not None
```

Comment:
since `nn_model_version` this will always be Falsey. changed the condition to `is not None`